### PR TITLE
prepare for RC2

### PR DESCRIPTION
**Pull request by Stebalien**
Not quite ready yet but might as well get this started.

I'm not going to maintain a changelog between these RCs (unless someone really objects). It's probably more work than it's worth.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # go-ipfs changelog
 
-## 0.4.18 2018-10-26-rc1
+## 0.4.18-rc2 2018-10-26
 
-This is probably one of largest go-ipfs release to date, 3 months in the making.
+This is probably one of the largest go-ipfs release in recent history, 3 months
```
Comment (magik6k) [inline CHANGELOG.md]: `releases`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -119,6 +120,12 @@ to 32 bytes in length into a CID, instead of writing an actual block. This
 should significantly reduce the size of filesystem trees with many empty
 directories and tiny files.
 
+#### IPNS
+
+You can now, finally, publish and resolve paths with namespaces *other* than
+`/ipns` and `/ipfs` through IPNS. Critically, IPNS can now be used with IPLD
+paths (paths starting with `/ipld`).
```
Comment (magik6k) [inline CHANGELOG.md]: `/ipld` paths are a feature of 0.4.18 too - https://github.com/ipfs/go-ipfs/pull/4672. Should I add a note about that?

**Comment by magik6k** [comment]
(1 minor nit)

File: CHANGELOG.md (hunk 2 above)
Comment (Stebalien) [reply CHANGELOG.md]: YES.

**Pull request merged**